apps/fz_http/priv/repo/seeds.exs

```diff
@@ -55,12 +55,6 @@ alias FzHttp.{Devices, ConnectivityChecks, Rules, Users}
     tx_bytes: 1_934_475_211_087_234
   })
 
-{:ok, _rule} =
-  Rules.create_rule(%{
-    device_id: device.id,
```

Comment:
@jamilbk judging by the code, a rule used to link to a device?

It is currently global (no longer has device_id) and the seeded rule blocks all outgoing traffic and is a major source of confusion that has caught all of us 😓 I hope removing it from seeds will be fine..?

Reply:
Yep! Sorry should have mentioned that earlier.